deploy/kubernetes/csi-snapshotter/rbac-csi-snapshotter.yaml

```diff
@@ -40,13 +40,13 @@ rules:
   - apiGroups: ["snapshot.storage.k8s.io"]
     resources: ["volumesnapshotcontents/status"]
     verbs: ["update", "patch"]
-  - apiGroups: ["groupsnapshot.storage.k8s.io"]
+  - apiGroups: ["groupsnapshot.storage.openshift.io"]
```

Comment:
can we add a additional files for this new controller. we dont need all the RBAC's please add only required RBAC for the VGS controller. we will include that in bundle and discard other RBAC so that the permission is minimal not everything

Reply:
This is also avoid having merge conflicts

Reply:
Done. PTAL